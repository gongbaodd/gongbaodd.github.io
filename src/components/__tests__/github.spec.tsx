import React from "react";
import { shallow } from "enzyme";
import GitHub from "../github";

test("gtihub", () => {
  const wrapper = shallow(<GitHub />);
  expect(wrapper).toMatchSnapshot();
});
