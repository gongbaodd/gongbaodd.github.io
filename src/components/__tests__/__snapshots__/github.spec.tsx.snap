// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gtihub 1`] = `
<GithubCorner
  ariaLabel="打开Github"
  bannerColor="deeppink"
  direction="right"
  href="https://github.com/gongbaodd/gongbaodd.github.io"
  octoColor="#fff"
  size={80}
/>
`;
